test/sharness/t0053-dag.sh

```diff
@@ -163,7 +163,20 @@ test_dag_cmd() {
 	'
 
 	test_expect_success "dag put with dag-pb works output looks good" '
-		printf $HASH > dag_put_exp &&
+		echo $HASH > dag_put_exp &&
+		test_cmp dag_put_exp dag_put_out
+	'
+
+	test_expect_success "dag put multiple files" '
+		printf {\"foo\":\"bar\"} > a.json &&
+		printf {\"foo\":\"baz\"} > b.json &&
+		ipfs dag put a.json b.json > dag_put_out
```

Comment:
Awesome!

Will test this week how many files it can get at once w/o choking. Thanks a lot for adding this!